### PR TITLE
Added a clickable link to get the formatted version.

### PR DESCRIPTION
**Pull request by dhylands**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # WoT Capability Schemas
 A collection of Web of Things capability schemas to be used as optional semantic markup in Web Thing Descriptions.
+
+View a formatted version [here](https://github.com/mozilla-iot/schemas.git).
```
Comment (benfrancis) [inline README.md]: The live version is hosted here https://iot.mozilla.org/schemas/

**Comment by mrstegeman** [comment]
What Ben said. I also added a URL for this repo, so if you go the main repo page, there is now a clickable link at the top.

File: README.md (hunk 1 above)
Comment (dhylands) [reply README.md]: That's the link I thought I was pasting, but apparently I screwed it up :)

**Comment by dhylands** [comment]
> What Ben said. I also added a URL for this repo, so if you go the main repo page, there is now a clickable link at the top.

Sorry - what do you mean by the "main repo"?

**Pull request merged**